users/vieting/experiments/switchboard/transducer/feat/experiments.py

```diff
@@ -277,6 +287,114 @@ def run_rasr_gt_stage1():
     return nn_system, report
 
 
+def run_scf_stage1():
+    ctc_alignment = get_ctc_alignment(ctc_alignment_model="scf", alignment_epoch=400)
+
+    gs.ALIAS_AND_OUTPUT_SUBDIR = "experiments/switchboard/transducer/feat/"
+    _, dev_corpora, _ = get_switchboard_data()
+    returnn_datasets = get_returnn_datasets_transducer_viterbi(context_window={"classes": 1, "data": 121})
+    returnn_datasets_hash_break = get_returnn_datasets_transducer_viterbi(
+        context_window={"classes": 1, "data": 121},
+        keep_hashes=False,
+    )
+    returnn_datasets_align_ctc = get_returnn_datasets_transducer_viterbi(
+        alignment=ctc_alignment,
+        features="waveform_pcm",
+    )
```

Comment:
You only need the alignment from the CTC model. In fact, your trainings are using the wrong alignment.

Reply:
I get that the returnn_datasets_hash_break  is not needed, but I am not using the correct one right?